server/services/admin.ts

```diff
@@ -168,7 +168,7 @@ export = ({ strapi }: StrapiContext): IServiceAdmin => ({
       .findMany({
         ...params,
         populate: {
-          authorUser: true,
+          authorUser: { populate: { avatar: true } },
```

Comment:
```suggestion
          authorUser: { populate: DEFAULT_POPULATE_CLAUSE },
```

Or something similar (`POPULATE_AVATAR`). That will DRY code. 🤷‍♂️